src/request.ts

```diff
@@ -938,17 +959,21 @@ class DatastoreRequest {
    *   const apiResponse = data[0];
    * });
    */
-  save(entities, gaxOptions?, callback?) {
+  save(entities: Entities, gaxOptions?: CallOptions):
+      void|Promise<google.datastore.v1.ICommitResponse>;
```

Comment:
I don't think we need the void here

Reply:
Getting the same error/problem as `delete` - I'll figure something out